check_sites.py

```diff
@@ -50,28 +82,25 @@ def main():
     cli_primaries = []
     input_prefix = ''
     with_diff = False
-    opts, _ = getopt.getopt(args, "i:p:", ["data_directory=", "with_diff", 
-                                         "primaries="])
+    with_format = True
+    opts, _ = getopt.getopt(args, "i:p:", ["data_directory=", "with_diff",
+                                         "with_format", "primaries="])
     for opt, arg in opts:
         if opt == '-i':
             input_file = arg
         if opt == '--data_directory':
             input_prefix = arg
         if opt == '--with_diff':
             with_diff = True
+        if opt == '--with_format':
```

Comment:
Bikeshedding, but it feels like we could make `with_format` a bit clearer. Maybe `strict_formatting` is clearer?